src/pages/Scheme/components/DistributionSettings/helpers.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  DistributionDefinitionTypes,
+  DistributionParameterTypes,
+  DistributionTypes,
+} from 'generated/graphql';
+import { toPairs } from 'lodash';
+import {
+  DistributionParameterPercentileRank,
+  DistributionSettingsParameters,
+  PercentileField,
+} from 'pages/Scheme/components/DistributionSettings/components/DistributionSettingsForm/types';
+import {
+  percentileFieldRankTypes,
+  percentileFieldTypes,
+} from 'pages/Scheme/components/DistributionSettings/constants';
+import distributionParametersMap from 'pages/Scheme/components/DistributionSettings/data';
+
+const isNumeric = (num?: string | number) =>
+  (typeof num === 'number' || (typeof num === 'string' && num.trim() !== '')) &&
+  !Number.isNaN(num as number);
```

Comment:
Если число может прийти строкой, то тогда обработка строчного типа неполная
```js
isNumeric('abc') // true
```

```suggestion
const isNumeric = (num?: string | number) =>
  (typeof num === 'number' || (typeof num === 'string' && num.trim() !== '')) &&
  !Number.isNaN(Number(num));
```

Reply:
Почему мы не можем использзовать `isFinite` из `lodash`, там точно есть какие-то специфические отличия, из-за которых нам необходима фунция `isNumeric`?